static/.htaccess

```diff
@@ -83,10 +83,39 @@ RewriteRule ^(.*)/(.+)/\2/?$ /$1/$2 [R=301,L]
 # disabled blog, redirect to index page.
 RewriteRule ^blog/ / [R=301,L]
 
-# Product links that I don't think we can ever remove
+# Product links that need to be preserved for Desktop Modeler, in perpetuity.
 RewriteRule ^docs/guides/migrating-from-Camunda-Platform/?(.*)$ /docs/guides/migrating-from-camunda-7/$1 [R=301,L]
 RewriteRule ^docs/reference/bpmn-processes/?(.*)$ /docs/components/modeler/bpmn/$1 [R=301,L]
 
+# URLs that need to be preserved for connector element templates, in perpetuity.
+RewriteRule ^docs/?$ / [R=301,L]
+RewriteRule ^docs/apis-clients/operate-api/?$ /docs/apis-tools/operate-api/overview/ [R=301,L]
```

Comment:
Non-blocking for this PR, but what's the plan when the Operate API is deprecated/removed and we only have the C8 API? Do we redirect to the C8 API overview page?